docker-compose.yml

```diff
@@ -124,7 +124,7 @@ services:
         volumes:
           - ddagent:/tmp/ddagent:rw
     testagent:
-        image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.16.0
+        image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.17.0
```

Comment:
Need to update this in order for the POST requests to `tracer_flare/v1` endpoint to properly be checked by the testagent, and return a 200